locale/CMakeLists.txt

```diff
@@ -1,12 +1,19 @@
 if (LEATHERMAN_USE_LOCALES)
     find_package(Boost 1.54 REQUIRED COMPONENTS locale system)
+    if (BOOST_STATIC AND LEATHERMAN_USE_ICU)
+        find_package(ICU)
```

Comment:
I had to alter this to the following to get it to work for me:

`find_package(ICU COMPONENTS i18n uc)`

Without this, the compile kept forgetting to add `-licu18n`, causing linker errors.